UltiSnips/elixir.snippets

```diff
@@ -0,0 +1,126 @@
+snippet if
+if ${1:condition} do
+	${2:expression}
+${3:end}
+endsnippet
+
+snippet unless
+unless ${1:condition} do
+	${2:expression}
+${3:end}
+endsnippet
+
+snippet else
+else
+	${1:expression}
+end
+endsnippet
+
+snippet iff
+if ${1:condition}, do: ${2:expression}${3:, else: ${4:expression}}
+endsnippet
+
+snippet unlesss
+unless ${1:condition}, do: ${2:expression}${3:, else: ${4:expression}}
+endsnippet
+
+snippet def
+def ${1:name}${2:(${3:parameters})}${4: when ${5:guard-condition}} do
+	${0:expression}
+end
+endsnippet
+
+snippet defp
+defp ${1:name}${2:(${3:parameters})}${4: when ${5:guard-condition}} do
+	${0:expression}
+end
+endsnippet
+
+snippet deff
+def ${1:name}${2:(${3:parameters})}, do: ${0:expression}
+endsnippet
+
+snippet defpp
+defp ${1:name}${2:(${3:parameters})}, do: ${0:expression}
+endsnippet
+
+snippet fn "inline function" s
+fn ${1:${2:args} }-> ${0:expression} end
+endsnippet
+
+snippet for
+for ${1:item} <- ${2:items}${3:, into: ${4:%{}}} do
+	${0:expression}
+end
+endsnippet
+
+snippet cond "cond do" s
+cond do
+	${1:pattern match} -> ${2:expression}${3:
+	->$4}
+end${0}
+endsnippet
+
+snippet rec "receive do" s
+receive do
+	${1:pattern match} -> ${2:expression}${3:
+	->$4}
+end${0}
+endsnippet
+
+snippet case "case do" s
+case ${1:expression} do
+	${2:pattern match} -> ${3:expression}${4:
+	->$5}
+end${0}
+endsnippet
+
+snippet try
+try do
+	${1:expression}
+rescue
+	${2:pattern match} -> ${3:expression}${4:
+	->$5}
+end${0}
+endsnippet
+
+snippet -> "case/cond pattern match" s
+${1:pattern match} -> ${2:expression}${3:
+->$4}
+endsnippet
+
+snippet @doc
+@doc """
```

Comment:
Sounds good. Could you either push another commit to this branch, or edit this commit to remove these ones that could be in the snippet format please?